proto/api/v1/payment.proto

```diff
@@ -50,6 +59,7 @@ message PaymentCustomer {
   Address address = 9;
   optional Coupon coupon = 10;
   string vat = 11;
+  string phone_number = 12;
```

Comment:
I think phone_number and vat should be optional, or we need to fill them with an empty string in some cases.